types/es-to-primitive/es2015.d.ts

```diff
@@ -0,0 +1,5 @@
+declare function ToPrimitive(
+	input: any,
+	hint?: Function
```

Comment:
not just any function, but only String or Number or Symbol.

Reply:
Well, the implemented algorithm doesn’t actually really care [what function is supplied](https://github.com/ljharb/es-to-primitive/blob/9c0dd264951a7cd5bbd8c5d1951843bf48351113/es2015.js#L47-L54), although [that might be incorrect](https://tc39.es/ecma262/#sec-toprimitive).

Reply:
yeah, there should be an assertion. Either way the types should enforce it.